### DISCUSSION TITLE
Unexpected error when signing release

### DISCUSSION THREAD

**Discussion by heshanthenura** [jreleaser/jreleaser#1605] [Q&A] [answered]
Any Idea about how to fix this error?
![image](https://github.com/jreleaser/jreleaser/assets/75155192/be9a1b51-b33a-4580-a88b-f429ea642ecc)

**Comment by aalmiray** [comment, marked as answer]
You may be using invalid GPG keys. Try generating a new key pair and give it another go.

**Reply by heshanthenura** [reply]
you mean using gpg right?

**Reply by heshanthenura** [reply]
Fixed the error.
